unison-src/new-runtime-transcripts/stm.output.md

```diff
@@ -134,7 +134,7 @@ display m n s =
   "mismatch: " ++ toText m ++ " + " ++ toText n ++ " /= " ++ toText s
 
 nats : [Nat]
-nats = [89,100,116,144,169,188,200,233,256,300]
+nats = [8,10,11,14,16,18,20,23,25,30]
```

Comment:
This makes the test run faster, but I don't really know what I'm doing here.

Reply:
/cc @dolio was there anything special about the original numbers?